compose/snippets/src/main/java/com/example/compose/snippets/landing/LandingScreen.kt

```diff
@@ -82,9 +84,14 @@ fun NavigationItems(navigate: (Destination) -> Unit) {
 
 @Composable
 fun NavigationItem(destination: Destination, onClick: () -> Unit) {
-    Button(
-        onClick = { onClick() }
+    Box(
```

Comment:
Why a box instead of a button here?

Reply:
It looks better than a list of buttons :)